scraper.go

```diff
@@ -370,6 +372,7 @@ func newAgentCollector(conf *Config, agentJobHistoryCache AgentJobHistoryCache,
 			if err != nil {
 				return err
```

Comment:
Should we log and continue here as well?

Reply:
For GoCD API errors I think it makes sense to bail and then let the next scrape cycle try again.

Reply:
sure. otherwise the risk can be to flood gocd with requests.